supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -0,0 +1,69 @@
+[[red-hat-enterprise-linux-conventions]]
+
+For documentation questions, contact rhel-docs@redhat.com.
+
+[discrete]
+[[clean-install]]
+==== clean install (noun)
+*Description*: A clean install removes all traces of the previously installed operating system, system data, configurations, and applications and installs the latest version of the operating system.
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: xref:upgrade[upgrade], xref:in-place-upgrade[in-place upgrade]
+
+[discrete]
+[[conversion]]
+==== conversion (noun)
+*Description*: An operating system conversion is when you convert your operating system from a different Linux distribution to Red Hat Enterprise Linux.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[in-place-upgrade]]
+==== in-place upgrade (noun)
+*Description*: During an in-place upgrade, you replace the earlier version with the new version without removing the earlier version first. The installed applications and utilities, along with the configurations and preferences, are incorporated into the new version.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:upgrade[upgrade], xref:clean-install[clean install]
+
+[discrete]
+[[migration]]
+==== migration (noun)
+*Description*: Typically, a migration indicates a change of platform: software or hardware. Moving from Windows to Linux is a migration. Moving a user from one laptop to another or a company from one server to another is a migration. However, most migrations also involve upgrades, and sometimes the terms are used interchangeably.
+
+*Use it*: with caution
+
+*Incorrect forms*:
+
+*See also*: xref:update[update], xref:upgrade[upgrade], xref:conversion[conversion][discrete]
+
+[[update]]
+==== update (noun)
+*Description*: Sometimes called a software patch, an update is an addition to the current version of the application, operating system, or software that you are running. A software update addresses any issues or bugs to provide a better experience of working with the technology. In RHEL, an update relates to a minor release, for example, updating from RHEL 8.1 to 8.2.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[upgrade]]
```

Comment:
It looks like this anchor name is already in use, can you change it to something else?

Reply:
Fixed